src/components/map/plugin/PluginMap.js

```diff
@@ -199,44 +186,51 @@ class PluginMap extends Component {
         // No data warning added to legend instead
         const alerts = []; // getMapAlerts(this.props); // ADDED
 
-        return !alerts.length ? (
-            <div ref={node => (this.node = node)} style={styles.map}>
-                {mapViews
-                    .filter(layer => layer.isLoaded)
-                    .reverse() 
-                    .map((config, index) => {
-                        const Overlay = layerType[config.layer] || Layer;
-
-                        return (
-                            <Overlay
-                                key={config.id}
-                                index={index}
-                                openContextMenu={this.onOpenContextMenu.bind(
-                                    this
-                                )}
-                                {...config}
-                                isPlugin={true}
-                            />
-                        );
-                    })}
-                {basemap.isVisible !== false && (
-                    <Layer key="basemap" {...basemap} />
-                )}
-                <ContextMenu
-                    position={position}
-                    feature={feature}
-                    onDrillDown={() => this.onDrill('down')}
-                    onDrillUp={() => this.onDrill('up')}
-                />
-            </div>
-        ) : (
-            <div style={styles.alerts}>
-                {alerts.map(alert => (
-                    <div key={alert.id} style={styles.alert}>
-                        <strong>{alert.title}</strong>: {alert.description}
+        return (
+            <MuiThemeProvider theme={muiTheme}>
```

Comment:
Hmmm... I'm not actually sure we want to include the theme in `PluginMap`.  There will be multiple instances if the parent app (dashboards) has one, and the parent won't be able to override anything in the theme that affects the map.  But maybe we want to explicitly theme it here?

Reply:
Removed theme again.